translations/README.german.md

```diff
@@ -0,0 +1,216 @@
+# Hasura GraphQL Engine
+
+[![Docs](https://img.shields.io/badge/docs-v1.0-brightgreen.svg?style=flat)](https://docs.hasura.io)
+[![CircleCI](https://circleci.com/gh/hasura/graphql-engine.svg?style=shield)](https://circleci.com/gh/hasura/graphql-engine)
+
+
+<a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
+<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://eepurl.com/dBUfJ5"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura GraphQL Engine ist ein blitzschneller GraphQL Server, welcher dir die Moeglichkeit bietet **sofort und in Echtzeit GraphQL APIs in Postgres**, mit [**webhook triggern**](../event-triggers.md) fuer Datenbank-events und [**remote schematas**](../remote-schemas.md) fuer Businesslogik.
+
+Hasura hilft dir GraphQL-Anwendungen basierend auf Postgres zu entwickeln oder schrittweise zu GraphQL zu wechseln fuer schon existierende Anwendungen die Postgres benutzen.
+
+Dazu mehr auf [hasura.io](https://hasura.io) und in den [Docs](https://docs.hasura.io).
+
+------------------
+
+![Hasura GraphQL Engine Demo](../assets/demo.gif)
+
+------------------
+
+![Hasura GraphQL Engine Realtime Demo](../assets/realtime.gif)
+
+-------------------
+
+## Features
+
+* **Entwickle maechtige Queries**: Built-in Filter, Paginierung, Suchmuster, Bulk-Inserts, Updates, Loeschen von Mutationen
+* **Echtzeit**: Konvertiere jede GraphQL Query zu einer Live-Query mit Hilfe von Subscriptions
+* **Merge Remote Schemas**: Zugriff auf individuelle GraphQL Schemas fuer Businesslogik mit einem einzigen GraphQL-Engine-Endpoint [**Read more**](../remote-schemas.md)
+* **Steuere Webhooks oder Severless-Funktionen**: Mit Postgres insert/update/delete Events ([read more](../event-triggers.md))
+* **Funktioniert mit bereits existierenden Live-Datenbanken**: Steuere eine bereits existierende Postgres Datenbank an um eine fertige GraphQL API zu erhalten
+* **Detailierte Zugriffssteuerung**: Dynamischer Zugriff der sich muehelos mit deinem Authentifizierungs System verbinden laesst (z.B.: auth0, firebase-auth)
+* **Hochperformant & wenig Speicherbedarf**: ~15MB docker image; ~50MB RAM @ 1000 req/s; multi-core aware
+* **Admin UI & Migration**: Admin UI & von Rails inspirierte Schemata-integratiomn
+* **Postgres** ❤️: Unterstuertzt Postgres Typen (PostGIS/geo-location, etc.), aendert Views zu *graphen*, loest gespeicherte Funktionen aus oder Prozesse mit Mutationen
+
+Mehr dazu auf [hasura.io](https://hasura.io) und in den [Docs](https://docs.hasura.io).
+
+## Inhaltsverzeichnis
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
+
+- [Quickstart:](#quickstart)
```

Comment:
Some links here are not working correctly.

Reply:
Thank you! Somehow I got that wrong but got it all working now!